<launch>
  <!-- 启动键盘控制节点 -->
  <node name="keyboard_control" pkg="mecanum_control" type="keyboard_control_node.py" output="screen"/>
</launch>
